addons/mrp/views/mrp_production_views.xml

```diff
@@ -432,7 +432,7 @@
                                 context="{'default_date': date_finished, 'default_date_deadline': date_deadline, 'default_location_id': production_location_id, 'default_location_dest_id': location_dest_id, 'default_state': 'draft', 'default_production_id': id, 'default_picking_type_id': picking_type_id, 'default_company_id': company_id, 'form_view_ref': 'stock.view_stock_move_operations'}"
                                 readonly="state == 'cancel' or (state == 'done' and is_locked)" options="{'delete': [('state', '=', 'draft')]}">
```

Comment:
@ticodoo last question I swear :) 
it should be okay then to add "no_create"
`options="{'delete': [('state', '=', 'draft')], 'no_create':[('state', '=', 'done')]}"`

Reply:
Does that work? I only see `'no_create': True`, `'no_create': False`, `'no_create': 1` in the code :thinking: Since the `product_id` is a mandatory and can't be set on new lines, it's effectively the same restriction so we're good leaving it as is imo